public/login.html

```diff
@@ -60,14 +60,14 @@
 
 <script>
   function login(e) {
-    e.target.action = 'https://api.core.ac.uk/login_check?continue=' + encodeURI(
+    e.target.action = 'https://api.dev.core.ac.uk/login_check?continue=' + encodeURI(
```

Comment:
It's a pain*. We will need to revert it back before release.

@mcancellieri can we deploy login to production and avoid this?

Reply:
Well, cookie will be set for api.core.ac.uk and not for api.dev.core.ac.uk.

Reply:
I know. I am pointing that at some point we will need to replace all occurs of `api.dev.` to `api.` to make the application production really what may (definitely will) be painful.